suites/google/googleDataAccessTest.js

```diff
@@ -75,31 +77,269 @@ BeforeSuite(async (indexd) => {
 });
 
 AfterSuite(async (fence, indexd, users) => {
-  console.log('Running usersync job');
-  bash.runJob('usersync', args = 'FORCE true');
   console.log('Removing indexd files used to test signed urls');
   await indexd.do.deleteFileIndices(Object.values(indexed_files));
 });
 
+Before(async (google, fence, users) => {
+  // Cleanup before each scenario
+  console.log('deleting keys for SA associated with users 0, 1 and user2...');
+  ['user0', 'user1', 'user2'].forEach(async(user) => {
+    const getCredsRes = await fence.do.getUserGoogleCreds(users[user].accessTokenHeader);
+    console.log(`Keys from ${user}: ${JSON.stringify(getCredsRes.access_keys)}`);
+    if (getCredsRes.access_keys.length > 0) {
+      let saName = getCredsRes.access_keys[0].name.split('/')[3];
+      console.log(`delete any existing keys for service account ${saName}`);
+      const dcfSaKeys = await google.listServiceAccountKeys('dcf-integration', saName);
+      console.log(`#### ##:' ${JSON.stringify(dcfSaKeys.keys)}`);
+      if (dcfSaKeys.keys) {
+        dcfSaKeys.keys.forEach(async (key) => {
+          console.log(`the following key will be deleted: ${key.name}`);
+          await google.deleteServiceAccountKey(key.name).then((deletionResult) => {
+            console.log(`deletionResult: ${JSON.stringify(deletionResult)}`);
+            if (deletionResult instanceof Error) {
+              console.log(`WARN: Failed to delete key [${key.name}] from Google service account [${saName}].`);
+	      } else {
+                console.log(`INFO: Successfully deleted key [${key.name}] from Google service account [${saName}].`);
+	      }
+          });
+        })
+      };
+    }
+  });
+});
+
 After(async (fence, users) => {
   // Cleanup after each scenario
-  const unlinkResults = Object.values(users).map((user) => fence.do.unlinkGoogleAcct(user));
+  const unlinkResults = Object.values(users).map(async(user) => {
+    await apiUtil.sleepMS(1 * 1000).then(() => {
```

Comment:
why the sleep? I _should_ be okay without it I think

Reply:
Yeah, that was mostly done based on dogmatic conjectures around what would happen in the background if Fence, Database, Google, et al. take too long to process this sort of command.

Again, I'm happy to remove it in a new PR but I'd like to see how this one behaves out in the wild.